src/components/AudioPlayer/AudioPlayer.module.scss

```diff
@@ -2,6 +2,10 @@
 @use "src/styles/constants";
 @use "src/styles/utility";
 
+.emptySpacePlaceholder {
+  height: calc(1 * constants.$audio-player-default-height);
```

Comment:
height: constants.$audio-player-default-height;

Reminds me of this meme 😂 

![image](https://user-images.githubusercontent.com/11590314/136653458-f39dd2d2-cdf0-4d33-92d2-49f500526cba.png)

Reply:
lol